graph_annotation.go

```diff
@@ -0,0 +1,18 @@
+package mackerel
+
+// GraphAnnotation parameters to post graph annotation
+type GraphAnnotation struct {
+	Service     string   `json:"service,omitempty"`
+	Roles       []string `json:"roles,omitempty"`
+	From        int64    `json:"from,omitempty"`
+	To          int64    `json:"to,omitempty"`
+	Title       string   `json:"title,omitempty"`
+	Description string   `json:"description,omitempty"`
+}
+
+// CreateGraphAnnotation create graph annotation
```

Comment:
[nit] If we follows go convention, comment sentences should be end with period.
rationale: https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

but, I think it's ok to ignore it.